### ISSUE TITLE
"Silicon was prevented from modifying apps on your Mac"

### ISSUE THREAD

**Issue by infinitepower18** [DigiDNA/Silicon#25]
I got this message when double clicking on an app in the list to open Finder with the app highlighted.

<img width="474" alt="Untitled" src="https://github.com/DigiDNA/Silicon/assets/44692189/3e4929a5-46fb-412a-a4e2-ec7667f3533b">

What was it trying to do here?

**Comment by macmade** [comment]
It was trying to show the app in a Finder window:  
https://github.com/DigiDNA/Silicon/blob/6223d8fd8d4eb3d41a42a73a23ef826604107e8c/Silicon/Classes/App.swift#L133

Are you on Sonoma?

**Comment by infinitepower18** [comment]
No, 13.5.2.

**Comment by macmade** [comment]
Anything related to Silicon in System Settings > Privacy?

**Comment by infinitepower18** [comment]
<img width="715" alt="Untitled" src="https://github.com/DigiDNA/Silicon/assets/44692189/78fbe31f-0260-48e1-8668-7b02e82a5511">

**Comment by macmade** [comment]
What was the app you’re clicked on? Was it inside another app?

**Comment by macmade** [comment]
I cannot reproduce the issue with the same settings on macOS 13.5.2.  
You're sure you're using a version downloaded from here?

**Comment by infinitepower18** [comment]
Yes, it was downloaded from here. I'm not able to reproduce it anymore as it only came up once.

**Comment by infinitepower18** [comment]
Yes it was downloaded from this GitHub. it seems to happen randomly...

https://github.com/DigiDNA/Silicon/assets/44692189/193dca4c-874f-41d0-baa8-e872b57007eb

**Comment by macmade** [comment]
I've been able to reproduce the issue and I've send a feedback to Apple, as this behavior is obviously incorrect: FB13194680.

Will probably never get fixed anyway...
